packages/deploy/deploy/300_catalyst/302_catalyst_setup.ts

```diff
@@ -27,8 +29,24 @@ const func: DeployFunction = async function (
     );
     log(`MINTER_ROLE granted to 0x803E1522e136121c058dc9541E7B3164957c200e`);
   }
+  // TODO END
+
+  // set catalyst on Royalty Manager
+  const catalyst = await deployments.get('Catalyst');
+  // TODO this should not be hardcoded here
+  const royaltyAmount = 500;
+  await catchUnknownSigner(
+    execute(
+      'RoyaltyManager',
+      {from: contractRoyaltySetter, log: true},
+      'setContractRoyalty',
+      catalyst.address,
+      royaltyAmount
+    )
+  );
+  log(`Catalyst set on RoyaltyManager with ${royaltyAmount} BPS royalty`);
```

Comment:
Doesn't this need RoyaltyManager as a dependency?

Reply:
It can but it doesn't have to in the current setup.

The catalyst deploys when `AssetCreate` script is running which is when both Asset and RoyaltyManager are already deployed.

But I will add it since if we for some reason change the structure it would cause issues.

Reply:
Also the setup runs after Cat Deploy which is dependent on the RoyaltyManager, but I 100% agree we should be listing all deps for clarity